src/sonic-py-common/sonic_py_common/multi_asic.py

```diff
@@ -0,0 +1,299 @@
+import glob
+import os
+
+from natsort import natsorted
+
```

Comment:
Remove this blank line, as these four imports are all "third-party".

Reply:
Fixed in the latest commit

Reply:
Sorry, I should have been more specific. `natsort` is a third-party package, not part of the standard library, so the empty line should be above it, not below. It should be as follows per [PEP8](https://www.python.org/dev/peps/pep-0008/#imports):

```
import glob
import os

from natsort import natsorted
from swsssdk import ConfigDBConnector
from swsssdk import SonicDBConfig
from swsssdk import SonicV2Connector

from .device_info import CONTAINER_PLATFORM_PATH
from .device_info import HOST_DEVICE_PATH
from .device_info import get_platform
```